source/postprocess/particle_distribution_statistics.cc

```diff
@@ -0,0 +1,360 @@
+                  standard_deviation_sum += pdf.standard_deviation;
+
+                  function_min_sum += pdf.min;
+                  function_max_sum += pdf.max;
+                  function_min_min = std::min(function_min_min, pdf.min);
+                  function_max_max = std::max(function_max_max, pdf.max);
+                }
+            }
+        }
+
+      // Get final values from all processors
+      const double global_standard_deviation_max = Utilities::MPI::max (standard_deviation_max, this->get_mpi_communicator());
+      const double global_standard_deviation_min = Utilities::MPI::min (standard_deviation_min, this->get_mpi_communicator());
+      const double global_cells_with_particles = Utilities::MPI::sum (cells_with_particles, this->get_mpi_communicator());
+      const double global_standard_deviation_sum = Utilities::MPI::sum (standard_deviation_sum, this->get_mpi_communicator());
+      const double global_standard_deviation_mean = global_standard_deviation_sum/global_cells_with_particles;
+      const double global_function_min_min = Utilities::MPI::min(function_min_min,this->get_mpi_communicator());
+      const double global_function_max_max = Utilities::MPI::min(function_max_max,this->get_mpi_communicator());
+
+      // Get the average of the functions max and min values
+      const double global_function_min_sum = Utilities::MPI::sum (function_min_sum, this->get_mpi_communicator());
+      const double global_function_max_sum = Utilities::MPI::sum (function_max_sum, this->get_mpi_communicator());
+      const double global_function_min_mean = global_function_min_sum/global_cells_with_particles;
+      const double global_function_max_mean = global_function_max_sum/global_cells_with_particles;
+
+      // Write to statistics file
+      statistics.add_value ("Minimum PDF standard deviation ", global_standard_deviation_min);
+      statistics.add_value ("Mean of PDF standard deviation: ", global_standard_deviation_mean);
+      statistics.add_value ("Maximum PDF standard deviation: ", global_standard_deviation_max);
+      statistics.add_value ("Mean of PDF minimum values: ", global_function_min_mean);
+      statistics.add_value ("Mean PDF maximum values: ", global_function_max_mean);
+      statistics.add_value ("Minimum of PDF minimum values: ", global_function_min_min);
+      statistics.add_value ("Maximum of PDF maximum values: ", global_function_max_max);
+
+
+      std::ostringstream output;
+      output << global_standard_deviation_min <<"/" << global_standard_deviation_mean <<"/" << global_standard_deviation_max <<", ";
+      output << global_function_min_mean <<"/" << global_function_max_mean << ", " << global_function_min_min << "/" << global_function_max_max;
+
+
+      return std::pair<std::string, std::string> ("Particle Distribution Statistics (standard deviation min/mean/max, mean of function min/max, absolute min/max):",
```

Comment:
```suggestion
      return std::pair<std::string, std::string> ("Particle distribution statistics (standard deviation min/mean/max, mean of function min/max, absolute min/max):",
```
This is a very long string, much longer than for the typical postprocessor. Could we shorten this? Perhaps replace "standard deviation" by "stddev" or something like this, given that we're already abbreviating min/max.